### PR TITLE
Add CreateCardForIssueFromLabel

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Migration of https://github.com/SonarSource/sonar-dotnet/blob/9071b131df2277c2a0d2b36ff1a2a97be5682ba0/.github/workflows/LabelIssue.yml

FIXME: 
* [ ] Needs JS rebuild

### CODE REVIEW

File: CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                await project.moveOrCreateCard(this.payload.issue as IssueOrPR, column.id);
+            }
+        } else {
+            this.log("Unexpected label name: " + labelName);
+        }
+    }
+}
+
+const action = new CreateCardForIssueFromLabel();
+action.run();
```
Comment (ilia-kebets-sonarsource) [inline CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts:24]: if `action.run()` is async, you should use an [IIFE](https://anthonychu.ca/post/async-await-typescript-nodejs/):

```javascript
(async () => {
     await action.run();
})();
```

File: CreateCardForIssueFromLabel/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# CreateCardForIssueFromLabel
+
+Create card for issue in a project column determined by the issue label.
```
Comment (ilia-kebets-sonarsource) [inline CreateCardForIssueFromLabel/ReadMe.md:3]: `Create **a** card for **an** issue in a project column determined by the issue label.`

If it works well on Windows, I would recommend you use Grammarly at OS level. It catches these missing articles most of the time.

**Review by ilia-kebets-sonarsource — APPROVED** [review]
I left 2 comments otherwise, LGTM!

File: CreateCardForIssueFromLabel/ReadMe.md (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply CreateCardForIssueFromLabel/ReadMe.md:3]: Sometimes it's over-adding those compared to native speakers :/

File: CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts:24]: I've made the `run` sync and used this IIFE trick inside. That keeps the boilerplate low.

Comment (pavel-mikula-sonarsource) [reply CreateCardForIssueFromLabel/CreateCardForIssueFromLabel.ts:24]: Reverted after discussion about GH actions behavior. 
>I can see it seems to be the rather proposed solution to use `action.run()` barely: https://github.com/actions/typescript-action

**Pull request merged**